hi-ml-histopathology/.mypy.ini

```diff
@@ -0,0 +1,12 @@
+[mypy]
+python_version=3.7
+scripts_are_modules=True
+namespace_packages=True
+show_traceback=True
+ignore_missing_imports=True
+follow_imports=normal
+follow_imports_for_stubs=True
+disallow_untyped_calls=False
+disallow_untyped_defs=True
+strict_optional=True
+mypy_path=$MYPY_CONFIG_FILE_DIR/src:$MYPY_CONFIG_FILE_DIR/../hi-ml/src:$MYPY_CONFIG_FILE_DIR/../hi-ml-azure/src:
```

Comment:
Does this mean that running mypy in hi-ml-histopathology will also run mypy on hi-ml and hi-ml-azure?

Reply:
no, these are the paths where mypy looks for source. And here we want to tell it about the existing of the other package code.